pkgs/development/haskell-modules/configuration-ghc-9.4.x.nix

```
@@ -59,6 +59,9 @@ self: super: {
   # Becomes a core package in GHC >= 9.10
   os-string = doDistribute self.os-string_2_0_7;
 
+  # Becomes a core package in GHC >= 9.10
```

Normally this means that we would need to provide a suitable version for lower GHC versions. Best to note why this isn't possible.

```suggestion
  # Becomes a core package in GHC >= 9.10, no release compatible with GHC < 9.10 is available
```